ban/db/fields.py

```diff
@@ -151,6 +151,21 @@ def coerce(self, value):
         return value
 
 
+class FantoirField(CharField):
+
+    def __init__(self, *args, **kwargs):
+        kwargs['max_length'] = 10
+        super().__init__(*args, **kwargs)
+
+    def coerce(self, value):
+        value = str(value)
+        if len(value) == 10:
+            value = value[:9]
+        if not len(value) == 9:
+            raise ValueError('Invalid fantoir "{}"'.format(value))
```

Comment:
I think we want a message explaining how to build the fantoir, something like:

`FANTOIR must be municipality INSEE + 4 first chars of FANTOIR'